engine/CMakeLists.txt

```diff
@@ -1370,13 +1370,13 @@ ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
                 ELSE (USE_PYTHON_3)
                     SET(CPACK_RPM_PACKAGE_REQUIRES "python27, boost-python2")
                 ENDIF (USE_PYTHON_3)
-            ELSE (LINUX_VERSION_ID VERSION_EQUAL 32)
```

Comment:
does this work? We're still using CMake 3.5 and the docs (https://cmake.org/cmake/help/v3.5/command/if.html) show that the `ELSE` and `ENDIF` need the expression from the first `IF` listed with them. It does seem later version simplified it and removed that requirement though.

Reply:
https://packages.ubuntu.com/xenial-updates/cmake shows Xenial (16.04) is still Cmake 3.5

Reply:
Fixed